components/public-api-server/pkg/apiv1/team.go

```diff
@@ -114,6 +118,68 @@ func (s *TeamService) ListTeams(ctx context.Context, req *connect.Request[v1.Lis
 	}), nil
 }
 
+func (s *TeamService) UpdateTeam(ctx context.Context, req *connect.Request[v1.UpdateTeamRequest]) (*connect.Response[v1.UpdateTeamResponse], error) {
```

Comment:
Unit tests, please. So far the Public API has nearly 100% coverage and I'd like to keep it that way.